File: blazor-wasm/Master-Detail/Models/Northwind/EmployeeType.cs
using Master_Detail.Models.Northwind.Models;

namespace Master_Detail.Models.Northwind
{
    public class EmployeeType
    {
        public int EmployeeID { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string Title { get; set; }
        public string TitleOfCourtesy { get; set; }
        public string BirthDate { get; set; }
        public string HireDate { get; set; }
        public AddressType Address { get; set; }
        public int ManagerID { get; set; }
        public string Notes { get; set; }
        public string AvatarUrl { get; set; }
    }
}

File: TestGlobalVarsNg/src/app/models/employee-type.ts
import { AddressType } from "./address-type";

export interface EmployeeType {
    "employeeID": number,
    "lastName": string,
    "firstName": string,
    "title": string,
    "titleOfCourtesy": string,
    "birthDate": string,
    "hireDate": string,
    "address": AddressType,
    "managerID": number,
    "notes": string,
    "avatarUrl": string
  }
  